README.md

```diff
@@ -333,7 +333,7 @@ To disable the version checks set the global `--check-version` flag to `false` (
 - `--hash-data` - Generate file data hashes (default: false).
 - `--detect-duplicates` - Detect duplicate files based on their hashes (default: false).
 - `--show-duplicates` - Show all discovered duplicate file paths (default: true).
-- `--detect-utf8` - Detect utf8 files and optionally extract the discovered utf8 file content (possible values: "true" or "dump" or "dump:output_target").
+- `--detect-utf8` - Detect utf8 files and optionally extract the discovered utf8 file content (possible values: "true" or "dump" or "dump:output_target.tgz" or "dump:output_target.tgz:max_size_bytes").
```

Comment:
We should leave room for the optional path pattern similar to what's done with the `--change-path` and `--change-data` parameters (ideally it'll be good to have a place for an optional content regex too), so we could have value formatted like this:

* `dump:<output type>:<path pattern>:<max size in bytes>` or
* `dump:<output type>:<path pattern>:<data regex>:<max size in bytes>`

Sample value would look like this: `dump:output_target.tgz::max_size_bytes` or `dump:output_target.tgz:::max_size_bytes`